src/main/webapp/app/pages/LevelOfEvidencePage.tsx

```diff
@@ -188,150 +223,190 @@ export default class LevelOfEvidencePage extends React.Component<
-                <span className={'d-flex align-items-center form-check'}>
-                  <Form.Group>
-                    {[Version.FDA, Version.AAC].map(versionCheck => (
-                      <Form.Check
-                        label={`Show mapping to ${LEVEL_NAME[versionCheck]}`}
-                        type="checkbox"
-                        onChange={() =>
-                          this.toggleVersion(
-                            this.version === versionCheck
-                              ? Version.V2
-                              : versionCheck
-                          )
-                        }
-                        name="mapping-to-other-levels"
-                        id={`mapping-to-other-levels-${versionCheck}`}
-                        key={`mapping-to-other-levels-${versionCheck}`}
-                        checked={this.version === versionCheck}
+    return (
+      <DocumentTitle title={getPageTitle(PAGE_TITLE[this.version])}>
+        <Row className="justify-content-center">
+          <Col lg={10}>
+            <div className="levels-of-evidence">
+              <Tabs
+                id="level-type-tabs"
+                onSelect={k => this.toggleVersion(Version[k || this.version])}
+                activeKey={this.activeKey}
+              >
+                {[Version.V2, Version.DX, Version.PX, Version.FDA_NGS].map(
+                  version => (
+                    <Tab
+                      eventKey={Version[version]}
+                      title={TAB_TITLES[version]}
+                      key={version}
+                    >
+                      <PageContent
+                        windowStore={this.props.windowStore}
+                        toggleVersion={this.toggleVersion}
+                        version={this.version}
+                        routing={this.props.routing}
                       />
-                    ))}
-                  </Form.Group>
-                </span>
-              )}
-            </Col>
-            <Col className={'col-auto'}>
-              {this.version !== Version.FDA_NGS && (
-                <Button
-                  size={'sm'}
-                  className={classnames('ml-1')}
-                  href={`content/files/levelOfEvidence/${this.version}/${
-                    LEVEL_FILE_NAME[this.version]
-                  }.ppt`}
-                >
-                  <i className={'fa fa-cloud-download mr-1'} />
-                  Download Slide
-                </Button>
-              )}
-              <DownloadButton
+                    </Tab>
+                  )
+                )}
+              </Tabs>
+            </div>
+          </Col>
+        </Row>
+      </DocumentTitle>
+    );
+  }
+}
+
+type PageContentProps = {
+  toggleVersion: (version: Version) => void;
+  windowStore: WindowStore;
+  version: Version;
+  routing: RouterStore;
+};
+
+class PageContent extends React.Component<PageContentProps> {
```

Comment:
We should move this to a functional component. If you need to do any work after component mount or when it updates, do that in LevelOfEvidencePage.tsx